components/HackneyDS/Select/index.js

```diff
@@ -26,7 +26,6 @@ const Select = (props) => {
     <div className="select-container">
       <select
         style={style}
-        disabled={isDisabled}
```

Comment:
why have you removed this?

Reply:
because we have disabled prop

Reply:
![image](https://user-images.githubusercontent.com/77606518/141445493-ee89d1af-dd78-415e-a5d3-620b5ac81838.png)

Reply:
but you're not passing 'disabled' into the select component?